### PR TITLE
Log acceptance tests to files

### PR DESCRIPTION
**Pull request by netoneko**
This PR drastically decreases memory usage of the acceptance suite, something that we initially believed was caused by a memory leak.

The actual problem was `go test` process capturing stdout and holding it in memory.

### CODE REVIEW

**Comment by netoneko** [comment]
Please don't merge this, there are some permissions issues while uploading acceptance logs.

File: test.acceptance.sh [hunk 1]
```diff
@@ -2,5 +2,5 @@
 
 source ./test.common.sh
 
-go test ./test/acceptance -count 100 -timeout 6m -failfast > test.out
+STANDALONE=true go test ./test/acceptance -count 100 -timeout 6m -failfast > test.out
```
Comment (talkol) [inline test.acceptance.sh]: `STANDALONE` is a very confusing name for this env variable. 
Maybe rename it to `NO_LOG_STDOUT`? It's a much clearer name that says exactly what it does..

**Comment by talkol** [comment]
In `test.acceptance.sh` we have this code:

```
go test ./test/acceptance -count 100 -timeout 6m -failfast > test.out
```

and in `test.common.sh` the function `check_exit_code_and_report` does:

```
grep -B 150 -A 15 -- "FAIL:" test.out > fail.out
        cat fail.out

        grep -B 150 -A 150 -- "test timed out" test.out > timed.out
        cat timed.out
```

This code greps on `test.out` which is redirection of STDOUT but we no longer log to STDOUT

File: test.acceptance.sh (hunk 1 above)
Comment (netoneko) [reply test.acceptance.sh]: Fixed

**Pull request merged**